app/util/networks/index.test.ts

```diff
@@ -339,4 +345,78 @@ describe('network-utils', () => {
       expect(releaseLockMock).toHaveBeenCalledTimes(1);
     });
   });
+  describe('convertNetworkId', () => {
+    it('converts a number to a string', () => {
+      expect(convertNetworkId(1)).toEqual('1');
+    });
+
+    it('converts a hexadecimal string to a decimal string', () => {
+      // Assuming convertHexToDecimal works correctly for '0x1' and returns 1
+      expect(convertNetworkId('0x1')).toEqual('1');
+    });
+
+    it('returns the same string if it is numeric', () => {
+      expect(convertNetworkId('123')).toEqual('123');
+    });
+
+    it('throws an error for non-numeric, non-hexadecimal strings', () => {
```

Comment:
You can't throw anything else than an error, so writing "throws an error" is redundant
```suggestion
    it('throws for non-numeric, non-hexadecimal strings', () => {
```

Reply:
Addressed on this [commit](https://github.com/MetaMask/metamask-mobile/pull/9065/commits/e4235d1ce48e74fe7ed729022dc8eace13ab5e44)